src/v/kafka/server/group_manager.cc

```diff
@@ -147,7 +147,7 @@ void group_manager::detach_partition(const model::ntp& ntp) {
             }
             ++g_it;
         }
-        _partitions.erase(it);
+        _partitions.erase(ntp);
```

Comment:
captured `ntp` is destroyed when co_await blocks?

Reply:
no, it will be destroyed after leaving the scope

Reply:
is this not subject to the capture-lifetime issue that exists with coroutines?

Reply:
`ntp` is a copy